packages/forma-36-codemod/bin/inquirer-choices.js

```diff
@@ -10,6 +12,16 @@ const PARSER_CHOICES = [
 ];
 
 const SETUP_CHOICES = [
+  {
+    name:
+      'run-all-v4: Update package json with new packages and remove old ones, remove v3 CSS imports and run all possible codemods for components.',
+    value: 'run-all-v4',
+  },
+  {
+    name:
+      'update-package-json: Updates package.json file with correct packages',
```

Comment:
I think this option should also remove CSS imports? Clean up of the package.json only does not help much I think? WDYT?

Reply:
true, if we are removing the packages from the packages.json there is no point in having the css still on the code

Reply:
cool! Now just the naming left I think. Maybe "update-setup" or something like this?